src/Generator/PropertyGenerator.php

```diff
@@ -136,43 +176,38 @@ public static function fromArray(array $array)
 
                     $property->setReadonly($value);
                     break;
+                case 'type':
+                    if (! $value instanceof TypeGenerator) {
```

Comment:
I'm so gonna deprecate all these `::fromArray()` horrors someday :D

Reply:
Never used the fromArray nor fromReflection methods anyway 😂